clear-intermediate-datasets/python-runnables/clear_int_datasets/runnable.py

```diff
@@ -76,6 +77,11 @@ def run(self, progress_callback):
         logging.info("Found {} FLOW OUTPUT datasets: {}".format(str(len(flow_outputs)),
                                                                 str(flow_outputs)))
 
+        # Identify Intermediate datasets:
+        intermediate_datasets = [x["name"] for x in all_datasets if x["name"] not in flow_inputs + flow_outputs]
+        logging.info("Found {} INTERMEDIATE datasets: {}".format(str(len(intermediate_datasets)),
+                                                                str(intermediate_datasets)))
+
         # Identify shared datasets:
         shared_objs = project.get_settings().settings["exposedObjects"]["objects"]
```

Comment:
shared_objs => shared_datasets

Reply:
Done in commit [892642f55ab9f78b502aec620fc31c0dc77d448c](https://github.com/gbetegon88/dataiku-contrib/commit/892642f55ab9f78b502aec620fc31c0dc77d448c#diff-6f25a7eaa37b1d34604450d8bc9334c0558917e69a3eb97e69bea9d70378e561).

Reply:
I think is shared_objs because not all inputs might be datasets (it could be a model for example). The next line is shared_datasets. I could change it for 'shared_objects' if it's more clear.

Reply:
good point ! shared_objects would be better indeed

Reply:
good point! shared_objects is more appropriate indeed

Reply:
Done in [af7c6256c6251eedf973804b221913910186eaea](https://github.com/gbetegon88/dataiku-contrib/commit/af7c6256c6251eedf973804b221913910186eaea#diff-6f25a7eaa37b1d34604450d8bc9334c0558917e69a3eb97e69bea9d70378e561)